addons/website_sale_product_configurator/controllers/main.py

```diff
@@ -14,10 +14,12 @@ def show_advanced_configurator_website(self, product_id, variant_values, **kw):
         """Special route to use website logic in get_combination_info override.
         This route is called in JS by appending _website to the base route.
         """
-        kw.pop('pricelist_id')
+        pricelist = kw.pop('pricelist_id')
```

Comment:
The pop was to prevent website users from being able to select an arbitrary pricelist. Use `website.get_current_pricelist()` instead